src/app/query/bitcoin/ordinals/inscriptions.query.ts

```diff
@@ -179,7 +179,7 @@ export function useGetInscriptionsInfiniteQuery() {
     staleTime: 3 * 60 * 1000,
     refetchOnMount: false,
     refetchOnReconnect: false,
-    refetchOnWindowFocus: false,
+    refetchOnWindowFocus: 'always',
```

Comment:
My bad no making a PR but I looked into this before.

Always is _way too often_. We need `true` here. The issue is that the data goes stale, and because of these 3 false values, it never refetches. If onWindowFocus is true, it'll refetch, but only when stale.

Reply:
Maybe I should reduce the `staleTime` here? Bc the prob is Mark reported the refetch isn't working, but that would be the case if `staleTime` is left at 3 min and the data isn't stale?

Reply:
Actually, maybe just instead of `refetchQueries` when the user clicks on the refetch icon we should actually use `invalidateQueries` which sets them as stale and refetches? 🤔

Reply:
I would leave it in explicitly, so we can set the default to false in future (we might need to to be more explicit)